sdk/logs/src/main/java/io/opentelemetry/sdk/logs/ReadWriteLogRecord.java

```diff
@@ -47,7 +53,87 @@ default ReadWriteLogRecord setAllAttributes(Attributes attributes) {
   /** Return an immutable {@link LogRecordData} instance representing this log record. */
   LogRecordData toLogRecordData();
 
-  // TODO: add additional log record accessors. Currently, all fields can be accessed indirectly via
-  // #toLogRecordData() at the expense of additional allocations.
+  /**
+   * Returns the value of a given attribute if it exists. This is the equivalent of calling
+   * getAttributes().get(key)
+   */
+  @Nullable
+  default <T> T getAttribute(AttributeKey<T> key) {
+    return toLogRecordData().getAttributes().get(key);
```

Comment:
Maybe it would be worth adding some unit tests for these default methods? It could be beneficial for possible future classes that do not overwrite them (or overwrite them partially)